rest-api-spec/src/main/resources/rest-api-spec/api/security.disable_user_profile.json

```diff
@@ -0,0 +1,41 @@
+{
+  "security.disable_user_profile":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-disable-user-profile.html",
+      "description":"Disable the user profile of the given unique ID so it is not visible to search."
+    },
+    "stability":"experimental",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_security/profile/{uid}/_disable",
+          "methods":[
+            "PUT",
+            "POST"
+          ],
+          "parts":{
+            "uid":{
+              "type":"string",
+              "description":"An unique identifier of the user profile"
```

Comment:
Do enable/disable_user_profile accept only one UID or multiple?

Reply:
A single one the same way as GetUserProfile API.